soql-analyzer/src/main/scala/com/socrata/soql/SoQLAnalyzer.scala

```diff
@@ -578,7 +578,8 @@ case class JoinAnalysis[ColumnId, Type](subAnalysis: Either[TableName, SubAnalys
     val (subAnasStr, aliasStrOpt) = subAnalysis match {
       case Right(SubAnalysis(subAnalysis, subAlias)) =>
         val selectStr = subAnalysis.toString
-        (s"($selectStr)", Some(subAlias))
+        if (subAnalysis.inParen) (s"$selectStr", Some(subAlias))
```

Comment:
This looks switched around

Reply:
Same explanation, it is something on top of the binaryTree.